anvil/Cargo.toml

```diff
@@ -35,7 +35,8 @@ features = [ "composite" ]
 gl_generator = "0.14"
 
 [features]
-default = [ "egl", "winit", "x11", "udev", "logind", "xwayland" ]
+#default = [ "egl", "winit", "x11", "udev", "logind", "xwayland" ]
+default = [ "egl", "x11" ]
```

Comment:
I don't understand this

Reply:
Probably testing is why most things are disabled temporarily

Reply:
Yeah, that is why this is still a draft PR. It is not completely ready ^^